pkg/daemon/daemon.go

```diff
@@ -70,6 +77,11 @@ func New(cfg *config.EnvConfig) (srv *Daemon, err error) {
 		})
 	})
 
+	staticDir := "/static/"
+	r.PathPrefix(staticDir).Handler(http.StripPrefix(staticDir, http.FileServer(http.Dir("."+staticDir))))
+
+	r.HandleFunc("/data", srv.dataHandler(engine)).Methods("GET")
```

Comment:
`/data` returns data visualised by dashboards. It returns CSV data for a single panel.